make EMULATOR=sims fail

Hello,
I'm trying to build the system with "make EMULATOR=sims", but after a while the build got a timeout with this error:

*:load sysbin;xxfile bin
*ttyop1$b   $g
$^K
*:PDUMP SYS2;TS XXFILE
DB     HACTRN SYS2   _DUMP_ OUTPUT WRITE  16:16:27
DB     HACTRN SYS2   TS     XXFILE DELRNM 16:16:27
*$P
$1B; TTYOP1>>SETZM TTYQ   kill
?U?

I'm using Ubuntu 18.04.

I did the same build under CentOS 6.9 and it worked.

regards,
Fausto

Tried the new branch... I got always a timeout but in a different place

```
(ASUM SUPRV COMM DSKFN MLISP MEDIT DB DISPLA GRAM GRIND COMPAR NFORMA TRIGI CSI!
MP MAT MATRIX FACTOR RAT3A RAT3B RAT3C RAT3D RAT3E NRAT4 LESFAC SPGCD SIMP INMI!
S OPERS MLOAD MERROR MFORMT ERMSGM RUNTIM UTILS MUTILS) 
(mapcan #'(lambda (x) (cond ((not (memq x
'(SETS TRANSS MTREE TRHOOK EDLM)
)) (doit x)))) (append todo todoi))
; Unfasl file ((DSK MUNFAS) ASUM UNFASL) not found.
; Unfasl file ((DSK MUNFAS) SERIES UNFASL) not found.
; Unfasl file ((DSK MUNFAS) NUMTH UNFASL) not found.
; Unfasl file ((DSK MUNFAS) SCHATC UNFASL) not found.
; Unfasl file ((DSK MUNFAS) TRGRED UNFASL) not found.
; Unfasl file ((DSK MAXOUT) COMBIN UNFASL) not found.
; Unfasl file ((DSK MUNFAS) SUPRV UNFASL) not found.
; Unfasl file ((DSK MUNFAS) COMM UNFASL) not found.
; Unfasl file ((DSK MAXOUT) COMM2 UNFASL) not found.
; Unfasl file ((DSK MUNFAS) DSKFN UNFASL) not found.
; Unfasl file ((DSK MUNFAS) MLISP UNFASL) not found.

The last command timed out.
Makefile:49: recipe for target 'out/sims/rp03.2' failed
make: *** [out/sims/rp03.2] Error 1
```
This seems a real timeout, maybe we should use a bigger global timeout.

Well, no dice.  I got:

> *:midas sys1;ts dir_bawden;dir^k
DB     MIDAS  SYS1   _MIDAS OUTPUT DELRNM 21:25:53
DB     MIDAS  SYS1   _MIDAS OUTPUT WRITE  21:25:53
DIR
DIR
Constants area inclusive
From    To
22327   22410
Run
DB     MIDAS  SYS1   TS     DIR    DELRNM 21:25:53
time = 0.00
1475 Symbols including initial ones (54% used)

:KILL
*
The last command timed out.
Makefile:46: recipe for target 'out/klh10/rp0.dsk' failed
make: *** [out/klh10/rp0.dsk] Error 1
jenkins@localhost:~/workspace/pdp-10-its-klh10$

--- a/build/build.tcl
+++ b/build/build.tcl
@@ -88,7 +88,7 @@ proc build_macsyma_portion {} {
     type "(mapcan #'(lambda (x) (cond ((not (memq x\r"
     type "'(SETS TRANSS MTREE TRHOOK EDLM)\r"
     type ")) (doit x)))) (append todo todoi))"
-    expect {
+    expect -timeout 1000 {
 	";BKPT" {
 	    type "(quit)"
 	}

